pandas/core/frame.py

```diff
@@ -4818,66 +4818,82 @@ def aggregate(self, func, axis=0, *args, **kwargs):
 
     def apply(self, func, axis=0, broadcast=None, raw=False, reduce=None,
               result_type=None, args=(), **kwds):
-        """Applies function along an axis of the DataFrame.
+        """
+        Apply a function along an axis of the `Series`.
```

Comment:
This is the dataframe docs, so Series -> DataFrame

(also it is not needed to put single backticks around Series and DataFrame, same for the rest of the docstring)

Reply:
add PR review suggestions from @jorisvandenbossche into commit c593a70c5